accord-core/src/main/java/accord/coordinate/AbstractCoordinatePreAccept.java

```diff
@@ -45,6 +47,8 @@
  */
 abstract class AbstractCoordinatePreAccept<T, R> extends SettableResult<T> implements Callback<R>, BiConsumer<T, Throwable>
 {
+    private static final Logger logger = LoggerFactory.getLogger(AbstractCoordinatePreAccept.class);
```

Comment:
you don't log anything, so this seems like dead code?

Reply:
I'll remove it